test/e2e/nexus_test.go

```diff
@@ -1,11 +1,12 @@
 package e2e
 
 import (
-	"github.com/m88i/nexus-operator/pkg/controller/nexus/resource/validation"
-	corev1 "k8s.io/api/core/v1"
 	"testing"
 	"time"
 
+	"github.com/m88i/nexus-operator/pkg/controller/nexus/resource/validation"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Have you been running [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) to sort these?

Right now [it's not a perfect drop-in replacement](https://github.com/golang/go/issues/21476) for gofmt, but what do you think of adding it to `hack/go-fmt`?

Deserves a separate issue/PR, but I was wondering.

Reply:
I think it will be fine to move.

Reply:
Created #171 :-)